From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Mon, 1 Feb 2016 12:09:34 +0100
Subject: tmpfiles: drop /run/lock/lockdev

Hardly any software uses that any more, and better locking mechanisms like
flock() have been available for many years.

Also drop the corresponding "lock" group from sysusers.d/basic.conf.in, as
nothing else is using this.

https://github.com/systemd/systemd/pull/2491
---
 sysusers.d/basic.conf.in | 1 -
 tmpfiles.d/legacy.conf   | 8 --------
 2 files changed, 9 deletions(-)

diff --git a/sysusers.d/basic.conf.in b/sysusers.d/basic.conf.in
index 823d6cb..b2dc5eb 100644
--- a/sysusers.d/basic.conf.in
+++ b/sysusers.d/basic.conf.in
@@ -19,7 +19,6 @@ g wheel   -     -            -
 
 # Access to certain kernel and userspace facilities
 g kmem    -     -            -
-g lock    -     -            -
 g tty     @TTY_GID@     -            -
 g utmp    -     -            -
 
diff --git a/tmpfiles.d/legacy.conf b/tmpfiles.d/legacy.conf
index 3cb0c63..62e2ae0 100644
--- a/tmpfiles.d/legacy.conf
+++ b/tmpfiles.d/legacy.conf
@@ -18,14 +18,6 @@ L /var/lock - - - - ../run/lock
 
 d /run/lock/subsys 0755 root root -
 
-# /run/lock/lockdev is used to serialize access to tty devices via
-# LCK..xxx style lock files, For more information see:
-# http://lists.freedesktop.org/archives/systemd-devel/2011-March/001823.html
-# On modern systems a BSD file lock is a better choice if
-# serialization is needed on those devices.
-
-d /run/lock/lockdev 0775 root lock -
-
 # /forcefsck, /fastboot and /forcequotacheck are deprecated in favor of the
 # kernel command line options 'fsck.mode=force', 'fsck.mode=skip' and
 # 'quotacheck.mode=force'
